### PR TITLE
Jd 2022 06 remove webpack library export

### PR DESCRIPTION
**Pull request by goapunk**

### CODE REVIEW

File: adhocracy-plus/assets/js/unload_warning.js [hunk 1]
```diff
@@ -26,11 +28,11 @@ $(function () {
   //   })
   // }
```
Comment (phillimorland) [inline adhocracy-plus/assets/js/unload_warning.js:29]: Now the embed is gone can we add this function back in?

**Review by phillimorland — COMMENTED** [review]
looks good and seems to work with uploads and category adding so should be all good, just couple comments

File: adhocracy-plus/assets/js/unload_warning.js [hunk 2]
```diff
@@ -3,16 +3,18 @@
...
     if (target.includes('dashboardToggle')) {
       submitted = true
     }
-    $(window).on('beforeunload', function (e) {
+    window.addEventListener('beforeunload', (e) => {
       if (!submitted) {
-        const string = django.gettext('If you leave this page changes you made will not be saved.')
+        // string is ignored on most modern browsers
+        const string = django.gettext('If you leave this page changes you made will not be saved. WTF')
+        e.preventDefault()
```
[the remark this answers is not in the document]
Comment (goapunk) [reply adhocracy-plus/assets/js/unload_warning.js:17]: ups :D

File: adhocracy-plus/assets/js/unload_warning.js (hunk 1 above)
Comment (goapunk) [reply adhocracy-plus/assets/js/unload_warning.js:29]: technically the embed isn't gone yet, is it?

Comment (goapunk) [reply adhocracy-plus/assets/js/unload_warning.js]: uncommented it, seems to work

**Review by phillimorland — APPROVED** [review]
very nice!

**Pull request merged**